### DISCUSSION TITLE
Start a search with an empty string

### DISCUSSION THREAD

**Discussion by aorith** [wezterm/wezterm#3046] [Q&A] [answered]
Hello,

Anyone knows if it's possible to always start a search with the prompt empty?
It really confuses me that the search is saved because I'm used to discarding it.

I've tried the following without success:

```lua
    {
      key = "f", mods = "SUPER",
      action = act.Multiple {
        act.ActivateCopyMode,
        act.CopyMode 'ClearPattern',
        act.Search { CaseInSensitiveString = '' },
      }
    },
```

It would be great if the search bar could be placed on top too, not sure if many people would be interested in that.

**Comment by wez** [comment, marked as answer]
Duplicate of:
* https://github.com/wez/wezterm/issues/1988
